src/constants/api.ts

```diff
@@ -5,3 +5,6 @@ export const getUserUrl = 'auth/me';
 export const refreshTokensUrl = 'auth/refresh';
 export const logoutUserUrl = 'auth/logout';
 export const getAllProductsUrl = 'product';
+// export const getAllProductsUrl = (minPrice: number, maxPrice: number): string => {
+//     return `product?minPrice=${minPrice}&maxPrice=${maxPrice}`
+// }
```

Comment:
Why commented?

Reply:
Wrong file version from this PR. Now it is changed.